QUARKUS-2727.md

```diff
@@ -0,0 +1,98 @@
+# QUARKUS-2736 - Encryption of configuration values
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-2727
+
+PR: https://github.com/quarkusio/quarkus/pull/31824 (https://github.com/smallrye/smallrye-config/pull/833 on SmallRye Config end)
+
+Upstream issue: https://github.com/quarkusio/quarkus/issues/31621
+
+Upstream documentation: https://quarkus.io/version/main/guides/config-reference#secret-keys-expressions
```

Comment:
That part is pretty small, pls ask for enhancement of that section

It's pretty important feature for production asked by other RH products too. So it's important to have more details available for smoother adoption.

Reply:
I'll open it when I'm finished with TD as then I'll have more things on my mind. I'll link it with this PR later today.